tests/unit/test_protobuf.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from jina.proto.jina_pb2 import Document
+from tests import JinaTestCase
+from google.protobuf.json_format import MessageToJson, Parse
+
+
+class ProtobufTestCase(JinaTestCase):
+
+    def test_tags(self):
+        d = Document()
+        d.tags['int'] = 1  # will convert to float!!!
```

Comment:
@JoanFM don't understand why but apparently this `Struct.Value` does not support `int`
 https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Value